frontend/templates/views/partials/analytics.j2

```diff
@@ -1,38 +1,150 @@
-          }
+    }
+  },
+  'triggers': {
+    'searchButton': {
+      'selector': '#searchTriggerOpen',
+      'on': 'click',
+      'vars': {
+        'event_name': 'search',
+        'event_category': 'search',
+        'event_action': 'open'
       }
-      {% endif %}
     }
   }
+} %}
+
+{% if '/documentation/examples/' in doc.pod_path %}
+{% set category = 'examples' %}
+{% elif '/documentation/components/' in doc.pod_path %}
+{% set category = 'components' %}
+{% elif '/documentation/courses/' in doc.pod_path %}
+{% set category = 'courses' %}
+{% elif '/documentation/guides-and-tutorials/' in doc.pod_path %}
+{% set category = 'guides-and-tutorials' %}
+{% endif %}
+
+{# On the courses page track how often the courses are downloaded #}
+{% if doc.pod_path == '/content/amp-dev/documentation/courses/teachers.md' %}
+{% do config['triggers'].update({'getCoursesButton': {
+  'selector': '#get-courses',
+  'on': 'click',
+  'vars': {
+    'event_name': 'courses',
+    'event_category': category,
+    'event_action': 'download'
+    }
+}}) %}
+{% endif %}
+
+{% if category %}
+{# Table of contents #}
+{% if not (doc.format.toc == '<div class="toc">\n<ul></ul>\n</div>\n' or not doc.format.toc) and not (doc.toc == False or doc.collection.toc == False) %}
+{# Track click on links inside the TOC #}
+{% do config['triggers'].update({'tocLink': {
+  'selector': '.ap-o-toc a',
+  'on': 'click',
+  'vars': {
+    'event_name': 'toc',
+    'event_category': category,
+    'event_action': 'link'
+    }
+}}) %}
+
+{# Track when the TOC is toggled #}
+{% do config['triggers'].update({'tocToggle': {
+  'selector': '.ap-o-toc label',
+  'on': 'click',
+  'vars': {
+    'event_name': 'toc',
+    'event_category': category,
+    'event_action': 'toggle'
+    }
+}}) %}
+{% endif %}
+
+{# Sidebar #}
+{# Track clicks on links inside the sidebar #}
+{% do config['triggers'].update({'sidebarLink': {
+  'selector': '.ap-o-sidebar a',
+  'on': 'click',
+  'vars': {
+    'event_name': 'sidebar',
+    'event_category': category,
+    'event_action': 'link'
+    }
+}}) %}
+{# Track when the sidebar is toggled #}
+{% do config['triggers'].update({'sidebarToggle': {
+  'selector': 'label[for="sidebar-desktop"], label[for="sidebar"]',
+  'on': 'click',
+  'vars': {
+    'event_name': 'sidebar',
+    'event_category': category,
+    'event_action': 'toggle'
+    }
+}}) %}
+
+{# Breadcrubms #}
+{# Track when a breadcrumb is used #}
+{% do config['triggers'].update({'breadcrumbLink': {
+  'selector': '.ap-m-breadcrumbs-crumb',
+  'on': 'click',
+  'vars': {
+    'event_name': 'breadcrumbs',
+    'event_category': category,
+    'event_action': 'link'
+    }
+}}) %}
+
+
+{# Format toggle #}
+{# Track when format is changed #}
+{% do config['triggers'].update({'formatToggle': {
+  'selector': '.ap-m-format-toggle-link',
+  'on': 'click',
+  'vars': {
+    'event_name': 'format-toggle',
+    'event_category': category,
+    'event_action': 'link'
+  }
+}}) %}
+
+{# Scroll #}
```

Comment:
We could move those out of the `{% if %}` as scroll depth is probably interesting for all our pages. @sebastianbenz?

Reply:
I agree, we should have this for all